modules/serverless-typescript-context-object-reference.adoc

```diff
@@ -0,0 +1,130 @@
+[id="serverless-typescript-context-object-reference_{context}"]
+= TypeScript context object reference
+
+The `context` object has several properties that can be accessed by the function developer.
+
+[id="serverless-typescript-context-object-reference-log_{context}"]
+== log
+
+Provides a logging object that can be used to write output to the cluster logs. The log adheres to the link:https://getpino.io/#/docs/api[Pino logging API].
+
+.Example log
+[source,javascript]
+----
+Function handle(context) {
+  context.log.info(“Processing customer”);
+  return 'OK';
+}
+----
+
+You can access the function by using the `curl` command to invoke it:
+
+.Example command
+[source,terminal]
+----
+$ curl http://example.com
```

Comment:
Should these examples be changed to use `kn func emit --sink local`?

Reply:
@lance Good idea. Could you please check all the substitutions I've made? 4a81bc1abdbf0805716b0490ea09be9e3910424e